src/components/WatchList/CoinChart.tsx

```diff
@@ -0,0 +1,157 @@
+import { Token } from '@pangolindex/sdk';
+import React, { useCallback, useContext, useState } from 'react';
+import { Link } from 'react-feather';
+import { Line, LineChart, ResponsiveContainer, Tooltip } from 'recharts';
+import { Box, Button, CurrencyLogo, Text } from 'src/components';
+import { ANALYTICS_PAGE, BETA_MENU_LINK, TIMEFRAME } from 'src/constants';
+import { useChainId } from 'src/hooks';
+import { Field } from 'src/state/pswap/actions';
+import { useSwapActionHandlers } from 'src/state/pswap/hooks';
+import { useTokenPriceData } from 'src/state/ptoken/hooks';
+import { formattedNum, toNiceDateYear } from 'src/utils/charts';
+import useUSDCPrice from 'src/utils/useUSDCPrice';
+import { unwrappedToken } from 'src/utils/wrappedCurrency';
+import { DurationBtns, SelectedCoinInfo, TrackIcons } from './styleds';
+
+type Props = {
+  coin: Token;
+};
+
```

Comment:
![Screenshot 2022-05-25 at 17-50-28 Pangolin](https://user-images.githubusercontent.com/6604146/170263518-e32111b2-f186-4332-b382-548c8250f6a7.png)

Check Chart is not working